extensions/amp-consent/validator-amp-consent.protoascii

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-consent
+  html_format: AMP
+  html_format: EXPERIMENTAL
```

Comment:
I believe this may be unintended and that this extension shouldn't be a part of EXPERIMENTAL format. (same for tags below)

Reply:
I saw a lot of tags are using the EXPERIMENTAL format : ) removed.